### DISCUSSION TITLE
where can i find an html5 build of kade engine where it's freeplay only

### DISCUSSION THREAD

**Discussion by dacixn** [KadeArchive/Kade-Engine#477] [Q&A]
so like in the main menu after pressing enter it says like only freeplay instead of storymode and freeplay

**Comment by puyoxyz** [comment]
you'll have to build it yourself, CI (automatic development builds) doesn't make builds like that

**Reply by dacixn** [reply]
crap

**Reply by dacixn** [reply]
> crap

i cant, i don't have the computer for it

**Reply by dacixn** [reply]
do you know where i can find a tutorial on how to compile the game on a linux machine?

**Reply by puyoxyz** [reply]
The website. https://kadedev.github.io/Kade-Engine/building
